.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules/
 *.vsix
 *.js
 *.taskkey
-*package-lock.json
\ No newline at end of file
+package-lock.json
+pullRequestInsightsTask/package-lock.json
```

Comment:
You shouldn't need to add this explicitly since it will be excluded from all subfolders as well.